resources/install.rb

```diff
@@ -28,12 +28,13 @@
 property :create_user, [true, false], default: true
 property :tmp_dir, String
 property :license, String, equal_to: ['accept']
+property :hab_version, String
 
 action :install do
   if ::File.exist?(hab_path)
     cmd = shell_out!([hab_path, '--version'].flatten.compact.join(' '))
     version = %r{hab (\d*\.\d*\.\d[^\/]*)}.match(cmd.stdout)[1]
-    return if version == hab_version
+    return if version == new_resource.hab_version
   end
 
   if platform_family?('windows')
```

Comment:
Was just thinking of this -- the install.rb is using curlbash for Linux install but we are doing Windows installation the "hard" way -- should we convert Windows installation to just use the iex method and pass versions if specified, similar to the curlbash for Linux installations?

Reply:
We probably should. I can do that now before we merge or we can follow up on it.

Reply:
We can do it in a follow-up, I think this one has enough changes =D